tests/acl/test_acl.py

```diff
@@ -1221,15 +1246,27 @@ def _verify_acl_traffic(self, setup, direction, ptfadapter, pkt, dropped, ip_ver
         testutils.send(ptfadapter, self.src_port, pkt)
         if direction == "uplink->downlink" and downstream_dst_port:
             if dropped:
-                testutils.verify_no_packet(ptfadapter, exp_pkt, downstream_dst_port)
+                try:
+                    testutils.verify_no_packet(ptfadapter, exp_pkt, downstream_dst_port)
+                except:
+                    debug_acl_failure(pkt, ip_version)
```

Comment:
@rajendrat We need to catch the exceptioni and raise it, otherwise the test case won't fail and always pass.